win32/README

```diff
@@ -0,0 +1,20 @@
+### Win32 build notes ###
+
+# Phil Ashby, Dec 2017
+
+Check config.mk for dependant packages. I recommend use of inline sed
+to fix up pkg-config file prefix settings once unpacked, then fix up and
+copy in gtk+-3.0.pc and gdk.pc if required.
+
+Compile tested on Debian 9 (stretch) using standard mingw-w64 toolchain.
+
+Sorry, no installer package as yet, you will need to copy out all the
```

Comment:
IMO we don't need any installer. A simple ZIP package like the ones we used to have seems sufficient and much easier to maintain.

Reply:
OK, Makefile could probably do with a 'dist' target then to put that together. On it.